docs-src/modules/developing/pages/views.adoc

```diff
@@ -141,106 +181,39 @@ session = LocalBuilder().build()
 view.delete_view("PurchaseStats")
 ----
 
+CLI::
++
 [source,bash]
-.with the CLI:
-----
-./kaskada-cli view delete -v PurchaseStats
-----
-
-== Managing views with the CLI and Spec files
-
-The CLI manages Kaskada resources declaratively by managing spec files.
-A spec file is a YAML file describing a set of Kaskada resources, for examples tables and views.
-
-=== Creating, updating and deleting views with the CLI
-
-
-The CLI uses a "declarative" API - rather than modifying resources directly you describe the desired state of Kaskada, and the `kaskada-cli sync` command makes whatever changes are needed to achieve this state.
-To create a view, add the view's description to your spec file and run `kaskada-cli sync apply`.
-To update a view, change the view's description in your spec file and run `kaskada-cli sync apply`.
-To delete a view, see xref:deleting-a-view[Deleting a view].
-
-=== The format of views in a spec file.
-
-Views are described in a spec file as a list of view objects under the `views` key:
-
-[source,yaml]
 ----
-views:
-      # A name used to refer to the view in queries
-    - view_name: PurchaseStats
-      # The expression to substitute anywhere the view's name is used
-      expression: |
-        {
-            time: Purchase.purchase_time,
-            entity: Purchase.customer_id,
-            max_amount: Purchase.amount | max(),
-            min_amount: Purchase.amount | min(),
-            count: CountPurchase,
-        } 
-      # A name used to refer to the view in queries
-    - view_name: CountPurchase
-      # The expression to substitute anywhere the view's name is used
-      expression: count(Purchase)
+kaskada-cli view delete PurchaseStats
 ----
+====
 
+A failed precondition error is returned if another view and/or
+materialization references the view. To continue with the deletion of
+the view, delete the dependent resources or supply the `force` flag to
+delete the view forcefully. Forcefully deleting a view without
+deleting the dependent resources may result in the dependent resources
+functioning incorrectly.
```

Comment:
```suggestion
delete the view forcefully. Forcefully deleting a view without
deleting its dependent resources may result in its dependent resources
functioning incorrectly.
```

Reply:
I might move the last sentence into an asciidoctor warning block
https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#admonitions